integration-test/1251-early-track-roads.py

```diff
@@ -0,0 +1,89 @@
+from . import FixtureTest
+
+
+class EarlyUnclassifiedRoads(FixtureTest):
+
+    def test_early_track_road_z11_grade1_paved(self):
+
+        # asphalt, grade1, track (default zoom 11, no demotion)
+        self.load_fixtures([
+            'https://www.openstreetmap.org/way/329375413',
+        ])
+
+        self.assert_has_feature(
+            11, 396, 781, 'roads',
+            {'kind': 'path',
+             'kind_detail': 'track'})
+
+    def test_early_track_road_z12_grade1_private(self):
+
+        # private, grade1, track (since private demoted from zoom 11)
+        self.load_fixtures([
+            'https://www.openstreetmap.org/way/10611894',
+        ])
+
+        self.assert_has_feature(
```

Comment:
Might it also be worth checking that there is no feature at zoom 11? Just in case we make a change later on which breaks the demotion.

For example, something like:

```python
self.assert_no_matching_feature(
  11, 330, 781, 'roads',
  {'kind': 'path', 'kind_detail': 'track'})
```

(With the fixtures, because we know they're the only features in the database, we don't need to specify `id`, but if you wanted to run this against the tiles then it would be harder, as we'd probably merge away the IDs by zoom 11. If we expose `private`, then that might be worth trying to match on?)